lebedigital/demonstrator_scripts/beam_design.py

```diff
@@ -150,15 +153,16 @@ def beam_section_design(
             S= min(0.25 * height, 200, slmax)
     #output final design
     out = {}
-    out["top_steel_dia[mm]"] = steel_dia
+    out["top_steel_dia[mm]"] = steel_dia * ureg('mm')
     out["top_steel_numbers"] = 2
-    out["shear_reinforcement_dia[mm]"] = 12
-    out["shear_reinforcement_spacing[mm]"] = S
-    out["required_area_bottom_steel[mm^2]"] = As1
+    out["shear_reinforcement_dia[mm]"] = 12 * ureg('mm')
+    out["shear_reinforcement_spacing[mm]"] = S * ureg('mm')
+    out["required_area_bottom_steel[mm^2]"] = As1 * ureg('mm^2')
 
     return out
 
 
+@ureg.check('[length]','[length]','[length]','[force]','[line_load]','[stress]','[stress]','[length]',None,'[length]')
```

Comment:
in the other merge request, you have used mass/time**2 for the line load, not sure which one is better, but I would suggest making it consistent.

Reply:
This is due to "work in progress" ;)
In "unit_registry.py" I defined my own line_load, as I did not find the mass/time**2 very intuitive. I will make it consistent.
Functionally it is the same.

Reply:
i changed it to [force]/[length], this does not require any new definition and is more intuitive